STACpopulator/log.py

```diff
@@ -144,6 +147,5 @@ def add_logging_options(parser: argparse.ArgumentParser) -> None:
     parser.add_argument("--debug", action="store_const", const=logging.DEBUG, help="set logger level to debug")
     parser.add_argument(
         "--log-file",
-        default=f"stac_populator_log_{dt.datetime.now(dt.timezone.utc).isoformat() + 'Z'}.jsonl",
```

Comment:
This line reminded me why `INFO` was used for `log-file` while `DEBUG` for `stderr`. The idea was that `log.info` + `extra_args` were used only for outputting STAC Items and relevant metadata, whereas `stderr` includes any other relevant debugging/step updates.

Now that `--log-level` would be used in the more "traditional" way of simply logging anything to file, maybe it is worth having an option to log only STAC Items metadata as `.jsonl` separately?